index.js

```diff
@@ -1,11 +1,12 @@
 const CloudFormationWrapper = require('./library/cloudformation');
 const S3Wrapper = require('./library/s3');
 
-function Rhinocloud() {
-  this.cloudformation = new CloudFormationWrapper();
-  this.s3 = new S3Wrapper();
+function Rhinocloud({ accessKeyId = undefined, secretAccessKey = undefined, region = undefined } = {}) {
```

Comment:
Shouldn't these default to `process.env.AWS_ACCESS_KEY_ID`, `process.env.AWS_SECRET_ACCESS_KEY`, and `process.env.AWS_REGION`?

Reply:
you actually don't have to do that - the sdk looks for the envs automatically.

https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-environment.html

Reply:
👍